api/swagger.yml

```diff
@@ -3457,6 +3457,10 @@ components:
           type: string
           description: Current version of the hermez-node
           example: v1.2.0
+        coordinatorBalance:
```

Comment:
Coordinators can use two addresses: one for forging and one for bidding. Most of the cases this address is the same, but not all the times. I'd suggest to:

a) Specify in the swagger doc that this address refers to the one used for forging (and probably change the name of `coordinatorBalance`) to help users understand
b) Add the bidder address as well